app/models/project_services/hipchat_service.rb

```diff
@@ -220,6 +233,10 @@ def create_note_message(data)
     message
   end
 
+  def notification?
+    self.notification == "1"
```

Comment:
Redundant `self` detected.<br>Prefer single-quoted strings when you don't need string interpolation or special symbols.